Jenkinsfile

```diff
@@ -16,293 +16,345 @@ pipeline {
         booleanParam(name: 'vendor', defaultValue: true, description: 'Vendor')
         booleanParam(name: 'windowsRS1', defaultValue: false, description: 'Windows 2016 (RS1) Build/Test')
         booleanParam(name: 'windowsRS5', defaultValue: false, description: 'Windows 2019 (RS5) Build/Test')
-  }
-  stages {
-    stage('Build') {
-      parallel {
-        stage('unit') {
-          when {
-            beforeAgent true
-            expression { params.unit }
-          }
-          agent { label 'amd64 && ubuntu-1804 && overlay2' }
-          environment { DOCKER_BUILDKIT='1' }
+    }
+    environment {
+        DOCKER_BUILDKIT = '1'
+        APT_MIRROR      = 'cdn-fastly.deb.debian.org'
+    }
+    stages {
+        stage('Build') {
+            parallel {
+                stage('unit') {
+                    when {
+                        beforeAgent true
+                        expression { params.unit }
+                    }
+                    agent { label 'amd64 && ubuntu-1804 && overlay2' }
 
-          steps {
-              sh '''
-                # todo: include ip_vs in base image
-                sudo modprobe ip_vs
+                    steps {
+                        sh label: 'Build dev image', script: '''
```

Comment:
If you're wanting to label these shell steps you could just stages within stages